test/test_synthetic.py

```diff
@@ -0,0 +1,407 @@
+    outputArrs = np.zeros((hgt_lvls.size, target_xyz[1].size, target_xyz[0].size))
+
+    # iterate over height levels
+    for hh, ht in enumerate(hgt_lvls):
+
+        llh = [xx, yy, np.full(yy.shape, ht)]
+
+        xyz = np.stack(lla2ecef(llh[1], llh[0], np.full(yy.shape, ht)), axis=-1)
+        LOS = los.getLookVectors(ht, llh, xyz, yy)
+
+        cos_factor = None
+
+        # 2d array where output is added to; one per hgt lvl
+        outSubs = outputArrs[hh, ...]
+        # iterate over all model levels
+        for zz in range(model_zs.size-1):
+            low_ht = model_zs[zz]
+            high_ht = model_zs[zz + 1]
+
+            if (high_ht <= ht) or (low_ht >= 50000):
+                continue
+
+            # If high_ht > max_tropo_height - integral only up to max tropo
+            if high_ht > 50000:
+                high_ht = 50000
+
+            # If low_ht < height of point - integral only up to height of point
+            if low_ht < ht:
+                low_ht = ht
+
+            # Continue only if needed - 1m troposphere does nothing
+            if np.abs(high_ht - low_ht) < 1.0:
+                continue
+
+            low_xyz = getTopOfAtmosphere(xyz, LOS, low_ht, factor=cos_factor)
+            high_xyz = getTopOfAtmosphere(xyz, LOS, high_ht, factor=cos_factor)
+
+            ray_length = np.linalg.norm(high_xyz - low_xyz, axis=-1)
+
+            outSubs   += ray_length
+
+            cos_factor = (high_ht - low_ht) / ray_length if cos_factor is None else cos_factor
+
+    return outputArrs
+
+
+@dataclass
+class StudyArea(object):
+    """ Object with shared parameters related to the study area """
+    reg:str
+    WM:str
+    wd = op.join(TEST_DIR, 'synthetic_test')
+
+    def __post_init__(self):
+        self.setup_region()
+
+        self.dts   = self.dt.strftime('%Y_%m_%d_T%H_%M_%S')
+        self.ttime = self.dt.strftime('%H:%M:%S')
+
+        self.wmObj = modelName2Module(self.WM.upper().replace("-", ""))[1]()
+
+        aoi = BoundingBox(self.SNWE)
+        aoi.add_buffer(buffer = 1.5 * self.wmObj.getLLRes())
+
+        self.los  = Raytracing(self.orbit, time=self.dt)
+
+        wm_bounds = aoi.calc_buffer_ray(self.los.getSensorDirection(), lookDir=self.los.getLookDirection())
+        self.wmObj.set_latlon_bounds(wm_bounds)
+        wm_fname  = make_weather_model_filename(self.WM, self.dt, self.wmObj._ll_bounds)
+        self.path_wm_real = op.join(self.wd, 'weather_files_real', wm_fname)
+
+        grid_spacing = 0.5
+        self.cube_spacing = np.round(grid_spacing/1e-5).astype(np.float32)
+        self.hgt_lvls     = np.arange(-500, 1500, 500)
+
+
+    def setup_region(self):
+        if self.reg == 'LA':
+            self.SNWE  = 33, 34, -118.25, -117.25
+            self.dt    = datetime(2020, 1, 30, 13, 52, 45)
+            self.orbit = self.wd + \
+                '/S1B_OPER_AUX_POEORB_OPOD_20210317T025713_V20200129T225942_20200131T005942.EOF'
+
+        elif self.reg == 'Fort':
```

Comment:
So for "region_short_name" or whatever you call it - you should have some documentation of what those regions are. Is it Los Angeles or Louisiana. Both are LA. Fort Lauderdale is under water right now, so that is the only Fort on my mind.